CHANGELOG.md

```diff
@@ -4,7 +4,21 @@ This file summarizes notable changes introduced in aktualizr version. It roughly
 
 Our versioning scheme is `YEAR.N` where `N` is incremented whenever a new release is issued. Thus `N` does not necessarily map to months of the year.
 
-## [??? (unreleased)]
+## [upcoming release]
+
+### Special considerations
+
+As a result of changes to the IP/POSIX Secondary protocol (see below), users of these Secondaries will need to take special care when upgrading their devices. The new version of aktualizr (e.g. the Primary) is backwards compatible and will work with both old and new versions of the protocol. However, the aktualizr-secondary is *not*. This means that if you are upgrading a device with IP/POSIX Secondaries, you should update the Primary ECU first, and if that is successful, then update your Secondaries.
+
+### Added
+
+- SetInstallationRawReport API call to enable setting a custom raw report field in the device installation result: [PR](https://github.com/advancedtelematic/aktualizr/pull/1628)
+- Allow ECU re-registration to support adding, removing, and replacing Secondaries: [PR](https://github.com/advancedtelematic/aktualizr/pull/1686)
+
+### Changed
+
+- Improved Secondary interface and error reporting, and improved IP/POSIX Secondary protocol: [PR](https://github.com/advancedtelematic/aktualizr/pull/1642)
```

Comment:
This is complicated, because there are three items that I'm trying to be clear about:
* the Secondary interface
* Secondary error reporting
* the IP/POSIX Secondary protocol

The problem is that the first two are general and affect all Secondaries, while the third does not. Maybe I'll just list them as two separate entries to make the distinction clearer.